src/gepa/api.py

```diff
@@ -102,7 +102,7 @@ def optimize(
     - perfect_score: The perfect score to achieve.
 
     # Component selection configuration
-    - module_selector: Component selection strategy. Can be a ReflectionComponentSelector instance, a string ('round_robin'), or None. If None, defaults to 'round_robin'. The 'round_robin' strategy cycles through components in order.
+    - module_selector: Component selection strategy. Can be a ReflectionComponentSelector instance, a string ('round_robin', 'all'), or None. If None, defaults to 'round_robin'. The 'round_robin' strategy cycles through components in order. The 'all' strategy selects all components for modification.
```

Comment:
Let's maybe replace None with 'round_robin'?

Reply:
"The 'all' strategy selects all components for modification in every GEPA iteration."

Reply:
sure!